.github/workflows/build-book-release.yml

```diff
@@ -134,7 +134,9 @@ jobs:
           ## Ideally, all dependencies should get installed in the first pass.
           # Remove when 0.60.1 available
           remotes::install_github("HenrikBengtsson/matrixStats", ref="develop")
-          remotes::install_github("biobakery/Maaslin2")  
+          remotes::install_github("biobakery/Maaslin2")
+          remotes::install_github("SchlossLab/mikRopML")
```

Comment:
I think this should be spelled "mikropml" (amd not mikRopML)?
See: http://www.schlosslab.org/mikropml/

Can you check and confirm.